### PR TITLE
feat: handleteamform trigger was moved after team verification

### PR DESCRIPTION
**Pull request by roger-in-kiva**

### CODE REVIEW

File: src/pages/Checkout/CheckoutPage.vue [hunk 1]
```diff
@@ -992,9 +992,6 @@ export default {
 						}
 					});
 				});
-				Promise.all(loans).then(() => {
```
Comment (mcstover) [inline src/pages/Checkout/CheckoutPage.vue:995]: Did you verify that the cache will update the the loan data after the mutation?

The end result should be that the Team Selection box for each Loan in the basket shows the newly joined team as selected.

File: src/pages/Checkout/CheckoutPage.vue [hunk 2]
```diff
@@ -879,6 +878,7 @@ export default {
 		},
 		verificationComplete() {
 			this.verificationSubmitted = true;
+			this.handleTeamForm();
```
Comment (mcstover) [inline src/pages/Checkout/CheckoutPage.vue:881]: This is probably a fine location to handle the "after" the verification process. However, we also need to check trigger the check for Team Join on page load IF verification isn't needed.

I suggest you add an else statement around line 865 in the `getPromoInformatinoFromBasket` method

File: src/pages/Checkout/CheckoutPage.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/Checkout/CheckoutPage.vue:995]: Do you mean this?
Screen Shot 2022-08-19 at 18 39 19

Comment (mcstover) [reply src/pages/Checkout/CheckoutPage.vue:995]: Correct. Just wanted to make sure the new team was selected after joining. You may be to remove your self from that team now to test this again...

Comment (roger-in-kiva) [reply src/pages/Checkout/CheckoutPage.vue:995]: Nice, I tested a couple of times adding and removing myself from that team...

File: src/pages/Checkout/CheckoutPage.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/pages/Checkout/CheckoutPage.vue:883]: An else statement was added to trigger the **handleTeamForm** method

File: src/pages/Checkout/CheckoutPage.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/Checkout/CheckoutPage.vue:995]: Should be great to test it on dev with more than one joined team to check if selecting as default

File: src/pages/Checkout/CheckoutPage.vue [hunk 3]
```diff
@@ -863,6 +862,8 @@ export default {
 						&& this.loans.length
 					) {
 						this.showVerification = true;
+					} else {
```
[the remark this answers is not in the document]
Comment (roger-in-kiva) [reply src/pages/Checkout/CheckoutPage.vue:865]: Right, they are in the method, are these correct? @mcstover 

```
handleTeamForm() {
			if (
				this.loans.length
				&& this.isActivelyLoggedIn
				&& this.teamId
				&& !this.teamJoinStatus
			) {
				// check for team join optionality
				this.showTeamForm = true;
			}
		}
```

[the remark this answers is not in the document]
Comment (mcstover) [reply src/pages/Checkout/CheckoutPage.vue:865]: oh ok thanks!

**Review by mcstover — APPROVED** [review]

**Pull request merged**